sdk/cosmos/microsoft-azure-cosmos-examples/src/main/java/com/azure/data/cosmos/examples/ChangeFeed/SampleChangeFeedProcessor.java

```diff
@@ -2,18 +2,8 @@
 // Licensed under the MIT License.
 package com.azure.data.cosmos.examples.ChangeFeed;
 
-import com.azure.data.cosmos.ChangeFeedProcessor;
-import com.azure.data.cosmos.ConnectionPolicy;
-import com.azure.data.cosmos.ConsistencyLevel;
-import com.azure.data.cosmos.CosmosClient;
-import com.azure.data.cosmos.CosmosClientException;
-import com.azure.data.cosmos.CosmosContainer;
-import com.azure.data.cosmos.CosmosContainerProperties;
-import com.azure.data.cosmos.CosmosContainerRequestOptions;
-import com.azure.data.cosmos.CosmosContainerResponse;
-import com.azure.data.cosmos.CosmosDatabase;
-import com.azure.data.cosmos.CosmosItemProperties;
-import com.azure.data.cosmos.SerializationFormattingPolicy;
+import com.azure.data.cosmos.*;
```

Comment:
We typically try to avoid wildcard imports. Certainly for actual implementation code we try this. Samples are less critical obviously, but it would be wise to not collapse needlessly.

Reply:
Replaced with single class imports